cmd/check_senders.cpp

```diff
@@ -646,7 +646,7 @@ int do_recover(app_options_t& options) {
         try {
             // Load collected data into Senders' table
             auto senders_table{lmdb_txn->open(db::table::kSenders)};
-            collector.load(senders_table.get(), nullptr, MDB_APPEND);
+            collector.load(senders_table.get(), nullptr, MDB_APPEND, "Senders");
```

Comment:
Is it necessary to define a label for connector ?
I believe a simple "Load progress %" is enough.
Nevertheless we might log an info line before invoking collector.load().

Reply:
We already do, in tx_lookup and blockhashes